pages/standards-guidelines/wcag/index.md

```diff
@@ -130,6 +130,13 @@ A few things have changed, and we intend the updates in the related documents to
 
 WCAG 2.0, WCAG 2.1, and WCAG 2.2 are all existing standards. WCAG 2.2 does not deprecate or supersede WCAG 2.1, and WCAG 2.1 does not deprecate or supersede WCAG 2.0. W3C encourages you to use the most recent version of WCAG.
 
+## Machine readable versions
+
+We maintain JSON versions of WCAG 2.1 and 2.2. The files include content and links for principles, guidelines, success criteria, techniques, and glossary terms in a hierarchical structure. These are updated monthly to ensure they align with the latest published version of the recommendation and related supporting materials.
+
+* [WCAG 2.1 JSON](https://www.w3.org/WAI/WCAG21/wcag.json)
+* [WCAG 2.2 JSON](https://www.w3.org/WAI/WCAG22/wcag.json)
+
```

Comment:
```suggestion

The JSON versions are governed by the legal framework of the WCAG standards, including W3C [liability](https://www.w3.org/policies/#Legal_Disclaimer), [trademark](https://www.w3.org/policies/#W3C_Trademarks) and [document use](https://www.w3.org/copyright/document-license/).
```

Since the JSON files do not include the legal information displayed in the documents on TR space, I suggest to clarify this in a new paragraph.

Reply:
I am wondering if a considerable amount of the legal disclaimer prose doesn't make as much sense for JSON resources (that do not themselves contain the legal disclaimer)?

Also, related to this:

> No material may be modified, edited or taken out of context such that its use creates a false or misleading statement or impression as to the positions, statements or actions of W3C.

In our last meeting we were also wondering if it would make sense to include remarks to the effect of making it very clear what information is coming from the WCAG data, vs. what data is added outside of it.